src/stores/store/keys/akv/akv.go

```diff
@@ -30,11 +28,7 @@ func New(client akv.KeysClient, logger log.Logger) *Store {
 	}
 }
 
-func (s *Store) Info(context.Context) (*entities.StoreInfo, error) {
-	return nil, errors.ErrNotImplemented
-}
-
-func (s *Store) Create(ctx context.Context, id string, alg *entities.Algorithm, attr *entities.Attributes) (*entities.Key, error) {
+func (s *Store) Create(ctx context.Context, id string, alg *entities.Algorithm, attr *entities.Attributes) (*models.Key, error) {
```

Comment:
IMO stores should always work with our entities, and models are a external reprensetation of our storage which never should leak into our application to reduce the potential impact of changes over our DB model

Reply:
moved to using models